shared/src/formationData.ts

```diff
@@ -349,14 +349,19 @@ export const nonprofitBusinessSuffix = [
   "INC.",
 ] as const;
 
-export const foreignCorpBusinessSuffix = [...corpBusinessSuffix, "P.C.", "P.A."] as const;
+export const foreignCorpBusinessSuffix = [...corpBusinessSuffix] as const;
+
+export const foreignCorpWillPracticeLawBusinessSuffix = [...corpBusinessSuffix, "P.C.", "P.A."] as const;
```

Comment:
Just wondering why you opted for const assertion vs. extending `corpBusinessSuffix` using interfaces and `extends`.

Reply:
So that only works with types and interfaces, the way this is being used requires an Array so it can use .map in the code base